otter/assign/utils.py

```diff
@@ -227,6 +227,7 @@ def run_generate_autograder(result, assignment, gs_username, gs_password, plugin
         # generate_args['token'] = token
         generate_args['course_id'] = str(pdf_args['course_id'])
         generate_args['assignment_id'] = str(pdf_args['assignment_id'])
+        generate_args['token'] = str(pdf_args['token'])
```

Comment:
Cannot assume that the `token` key will be provided. The default behavior should still be available if, e.g., the user does something like:

```yaml
generate:
    pdfs:
        course_id: 123456
        assignment_id: 456789
```

Reply:
updated, not sure why but .get() only seems to work if colon is included as part of the key value